README.adoc

```diff
@@ -285,6 +285,102 @@ spec:
 ...
 ----
 
+== Configure Health
+
+=== Kiali defaults
+
+By default Kiali health is:
+
+* Degraded when Error Rate is >= 0.1%
+* Failure when Error Rate is >= 20%
+
+For Kiali an error request is when:
+
+* For http the code status match the expression */^[4-5]\d\d$/*
+* For grpc the code status match the expression */^ [1-9]$|^1[0-6]$/*
+
+=== Configuration
+
+Kiali works with a FIFO configuration, so Kiali will use the first configuration that applies with the resource requested.
+
+So the granularity and criteria should be configured from highest to lowest.
+
+The health configuration follow this scheme:
+
+[source,yaml]
+----
+health_config:
+  rate:
+    - namespace: string
+      kind: string
+      name: string
+      tolerance:
+        - protocol: string
+          direction: string
+          code: string
+          degraded: number
+          failure: number
+          ...
+    ...
+
+
+----
+
+For each Rate Kiali will use the first match by:
+
+* Namespace (string) is a regular expression that match the namespace name (bookinfo, istio-system...)
+* Kind (Optional) (string) is a regular expression that match the resource type (app/workload/service)
```

Comment:
I think I missed the discussion about filtering by "kind".
I see that's optional but I think this can introduce more edges and corner cases rather than really help
(i.e. app is a composition of workload + services, if it has a different config, perhaps user can get easily some discrepancies, "apps are health, but services look in red").

I will continue looking the details and examples to add more comments about this.

Reply:
is a regex related with the resource